.eslintrc.json

```diff
@@ -12,7 +12,7 @@
     "@typescript-eslint/no-unused-vars": ["error", {
       "vars": "all",
       "args": "after-used",
-      "ignoreRestSiblings": false
+      "ignoreRestSiblings": true
```

Comment:
Useful for emulating `_.pick` / `_.omit` via destructuring, as done here:

https://github.com/electron/electron/pull/21236/commits/051706e4a8b620f46b3e7aa1f2096806e3b1653f#diff-39ff6eb84462bf6b40517f852555ac6dR262